Resources/Prototypes/Entities/Constructible/Ground/crayondecals.yml

```diff
@@ -0,0 +1,15 @@
+- type: entity
+  abstract: true
+  id: CrayonDecal
+  name: crayon drawing
+  description: "Graffiti. Damn kids."
+  components:
+  - type: Clickable
+  - type: InteractionOutline
+  - type: Collidable
+  - type: Sprite
+    sprite: Constructible/Misc/crayondecals.rsi
+    state: corgi
+  - type: Appearance
+    visuals:
+    - type: CrayonDecalVisualizer
```

Comment:
Should decals be their own entities? Seems like it could get expensive if people draw a lot. I think swept / pjb mentioned it too.

https://discordapp.com/channels/310555209753690112/310555209753690112/759719183415771147

IMO it should probably use chunking like the gas tile overlays.